src/Geno/C++/GUI/Widgets/WorkspaceWidget.cpp

```diff
@@ -167,8 +171,13 @@ void WorkspaceWidget::Show( bool* p_open )
 				{
 					ImGui::CloseCurrentPopup();
 
-					if( Project* prj = workspace->ProjectByName( selected_project_ ) )
+					if (Project* prj = workspace->ProjectByName( selected_project_ ))
```

Comment:
I know I'm being maticulous, but please fix the formatting. k thx 💋

Reply:
Oh dang didn't even see that